tests/test_peer_did_utils.py

```diff
@@ -0,0 +1,46 @@
+from peerdid.peer_did_utils import _encode_service
+
+
+def test_encode_service():
+    service = '''{
+        "type": "didcommmessaging",
+        "serviceEndpoint": "https://example.com/endpoint",
+        "routingKeys": ["did:example:somemediator#somekey"]
+        }
+        '''
+
+    assert _encode_service(
+        service) == ".SeyJ0IjoiZG0iLCJzIjoiaHR0cHM6Ly9leGFtcGxlLmNvbS9lbmRwb2ludCIsInIiOlsiZGlkOmV4YW1wbGU6c29tZW1lZGlhdG9yI3NvbWVrZXkiXX0="
+
+
+def test_encode_service_without_routing_keys():
+    service = '''{
+        "type": "didcommmessaging",
+        "serviceEndpoint": "https://example.com/endpoint",
+        }
+        '''
+    assert _encode_service(
+        service) == ".SeyJ0IjoiZG0iLCJzIjoiaHR0cHM6Ly9leGFtcGxlLmNvbS9lbmRwb2ludCIsfQ=="
+
+
+def test_encode_service_with_multiple_entries_list():
+    services = ['''
```

Comment:
Let's re-write the test by passing a single JSON string consisting of two service elements. Only one .S should be expected.